tests/e2e/scenarios/test_reborn_webui_v2_notifications.py

```
@@ -108,36 +171,133 @@ async def test_reborn_v2_notification_popover_opens_automation_thread(
+    context = await reborn_v2_browser.new_context(viewport={"width": 1280, "height": 720})
+    page = await context.new_page()
+    try:
+        await install_fake_v2_event_stream(page)
+        state = await _route_notification_inbox(
+            page,
+            notification_id=COMPLETION_NOTIFICATION_ID,
+            kind="run_completed",
+            turn_run_id=RUN_ID,
+        )
+
+        async def timeline_handler(route):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `timeline_handler`

Add return type annotation: `None`

(ANN202)